sdk/core/core-auth/src/tokenCredential.ts

```diff
@@ -15,7 +15,10 @@ export interface TokenCredential {
    * @param options The options used to configure any requests this
    *                TokenCredential implementation might make.
    */
-  getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
+  getToken(
+    requestContext: TokenRequestContext,
```

Comment:
In every instance of this PR where `getToken` has been updated to replace `scopes` with `requestContext`, the docs for the parameter should be updated too. They are currently referring to `scopes`